OWML.Launcher/App.cs

```diff
@@ -21,6 +21,8 @@ public class App
         private readonly OWPatcher _owPatcher;
         private readonly VRPatcher _vrPatcher;
 
+        private const string VrArgument = " -vrmode openvr";
```

Comment:
maybe use uppercase instead of pascal case? doesn't really matter that much, but upper case makes it clearer it's a constant

Reply:
in C#, PascalCase is normally used for consts

Reply:
commonly only defines are fully uppercase